vendor/github.com/openshift/cluster-api/pkg/drain/drain.go

```diff
@@ -334,6 +334,12 @@ func getPodsForDeletion(client kubernetes.Interface, node *corev1.Node, options
 	}
 
 	for _, pod := range podList.Items {
+		if !pod.ObjectMeta.DeletionTimestamp.IsZero() && time.Now().Sub(pod.ObjectMeta.GetDeletionTimestamp().Time).Minutes() > 1 {
```

Comment:
This seems to work, but for some reason, we appear to be processing the deletes serially.

Reply:
https://gist.github.com/michaelgugino/ebb5b60f6e1193d93b5f55a96e5c92e5